CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,207 @@
+
+### Examples
+- **Interim Member**: Has fewer than 8 substantive contributions but commits to achieve them within 3 months.
+- **Interim Reviewer**: Has not yet reviewed 20+ PRs but is actively being mentored to do so.
+
+### Process
+1. Maintainers discuss and vote on the need for an interim role (lazy consensus).
+2. Scope and duration are defined clearly (specialty, responsibilities, expected milestones).
+3. Nomination of interim roles is done by lazy consensus.
+4. Interim roles are granted for a maximum of three months.
+5. Interim status is reviewed at the end of the period:
+   - If requirements are met → promotion to the permanent role.
+   - If not met → revert to previous role; contributor may try again later.
+
+### Handling abuse
+
+Interim roles are not a substitute for permanent roles. If a contributor is abusing the interim role system, they may be demoted to their previous role.
+
+---
+
+## Advancement Process
+
+1. **Nomination** by an eligible community member (Member+).
+2. **Sponsorship** by two role holders at the **target level or higher** (within the specialty where applicable).
+3. **Review** of activity and behavior (quality, reliability, collaboration, responsiveness).
+4. **Decision** by lazy consensus of the relevant group (or **supermajority** if contested).
+
+---
+
+## Inactivity
+
+A role holder may be considered inactive if they have not actively contributed in their specialty or general project responsibilities for **six (6) consecutive months**.
+
+### Measurement Sources
+- GitHub activity: merged PRs, PR reviews, issue triage/comments.
+- Participation in community calls or asynchronous design discussions.
+- Specialty-specific metrics (e.g., CI job maintenance, provider updates, security disclosures).
+
+### Triggering Process
+1. **Detection**  
+   - Activity is reviewed at least quarterly by Maintainers or via automation.  
+   - Any Maintainer may propose an inactivity review for a role holder.
+2. **Notification**  
+   - A public issue is opened in a `community`/`governance` space (or email if sensitive).  
+   - The individual is tagged/emailed and given **30 days** to respond.
+3. **Grace Period**  
+   - If the contributor indicates intent to return, no change is made.  
+   - If there is no response or no significant activity within the grace period, proceed.
+4. **Decision**  
+   - Demotion is decided by **lazy consensus** of Maintainers, or **supermajority** if contested.
+5. **Documentation**  
+   - Update `OWNERS`, GitHub teams, and governance records.  
+   - Former Maintainers are listed as **Emeritus**.
+
+### Reinstatement
+A contributor can be reinstated at their previous level via the standard advancement process. Prior history is considered favorably.
+
+---
+
+## Conduct & CLA
+
+All contributors must follow the CNCF Code of Conduct and sign the CNCF CLA (where applicable) before contributions are merged.
```

Comment:
The CoC defines the rules for language and behaviour and the incident process.
But whats missing is handling abuse of privileges (e.g. installing crypto miner, approving malicious code, force pushing released image with malware etc.).

That's hard to define but i think these criteria and the process should be defined. :policeman: